graviti/portex/base.py

```diff
@@ -98,7 +152,7 @@ def from_pyobj(
         if _imports is None:
             _imports = Imports.from_pyobj(content.get("imports", []))
 
-        class_ = _imports[content["type"]]
+        class_: Type[_T] = _imports[content["type"]]  # type: ignore[assignment]
```

Comment:
where this ignore comes from?

Reply:
mypy considers that the expression has type "Type[PortexType]".